src/main/java/frc/robot/Robot.java

```diff
@@ -98,6 +98,7 @@ public class Robot extends LoggedRobot {
 
     private final Trigger autoEnabled = new Trigger(DriverStation::isAutonomousEnabled);
     private final Trigger teleopEnabled = new Trigger(DriverStation::isTeleopEnabled);
+    private final Trigger endgameTrigger = new Trigger(() -> DriverStation.getMatchTime() <= 20).and(teleopEnabled);
```

Comment:
This probably should either check if the match time is `!= 0` (or whatever `getMatchTime` reports without a FMS) or check if an FMS is connected - cause atm, this will rumble the controller whenever you enable teleop without an FMS.

Reply:
FMS connected seemed like the best way to do this, even if we *are* giving up the ability to test this without an FMS connection (such as in practice mode). The behavior of `getMatchTime()` varies too much to cover every case, checking if FMS existed seemed like the easiest and least complex way that did not involve huge trade-offs.